sysconfigs/daint/compilers.yaml

```diff
@@ -89,21 +89,6 @@ compilers:
     - pgi/20.1.0
     environment: {}
     extra_rpaths: []
-- compiler:
```

Comment:
wont the change in this file have side effects on all the other spack package on daint ?

Reply:
It will, but it is a desired change.